lib/commands/moveStalledJobsToWait-7.lua

```diff
@@ -62,6 +62,7 @@ if(#stalling > 0) then
         if(stalledCount > MAX_STALLED_JOB_COUNT) then
           rcall("ZADD", KEYS[4], ARGV[3], jobId)
           rcall("HSET", jobKey, "failedReason", "job stalled more than allowable limit")
+          rcall("PUBLISH", KEYS[4], jobId)
```

Comment:
I think it would be better to create a json object in lua, its pretty easy (untested):;
```lua
rcall("PUBLISH", KEYS[4],  "{\"jobId\":\"" .. jobId .. "\", \"val\": \"job stalled more than maxStalledCount\"}")
```
That will also eliminate the special case handling you added in queue.js.